ads/vendors/myfinance.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import {validateData, writeScript} from '../../3p/3p';
+import { validateData, writeScript } from '../../3p/3p';
```

Comment:
nit: prettier is going to complain about this, you can try `amp lint --local_changes --fix` or follow or docs to setup the formatting extensions in your editor.

Reply:
Fixed.